ansible/vars.yml

```diff
@@ -17,6 +17,12 @@ load_balancer_port: 8443
 # Default: "weave"
 cni: weave
 
+# Configure Kubernetes CRI
+# Options: "containerd" or "Docker"
```

Comment:
```suggestion
# Options: "containerd" or "docker"
```

I'd prefer lowercase, although if we use uppercase it should be consistent either way. Small change.